### ISSUE TITLE
Extra information sources

### ISSUE THREAD

**Issue by lynxlynxlynx** [alexbatalov/iwd2-re#1]
Hey, just noticed this due to the recent PortMaster Fallout ports. If this project is still alive, you can gain a lot of info from:
- [IESDP](https://gibberlings3.github.io/iesdp/index.htm), where for example file formats are documented
- [GemRB](https://gemrb.org/), where the code does not try to copy verbatim the original, but is indeed also a result of a lot of binary reverse engineering over the years — it would be impossible without it
- there have been a few DLL-injected enhancers for iwd2 as well, but I'm not sure if much can be learned from them for an effort like this

**Comment by alexbatalov** [comment]
Yep, still alive. I've stepped back for a couple of months to finish Fallout, but plan to get back to it in the future. Thanks for your links, will look into (I guess I've already used IESDP for some of structs).

**Comment by lynxlynxlynx** [comment]
What's the general idea though, is it just an exercise for you?

**Comment by alexbatalov** [comment]
Kind of. With Fallouts 1 & 2 I know it is possible to reverse engineer entire game from start to finish. This time the scope is much larger, the language is different, and I want to measure how much it will actually take. When completed it will be the biggest reverse engineering project of all time. Sounds good enough for me to try.

**Comment by lynxlynxlynx** [comment]
We've already done most of it in GemRB, but the old IDA files have been lost (worked only for the old free version anyway). You can also look at Eeex's pdbs, since iwd2 is similar to older/newer engines in many regards. There's also a fan-driven iwd2ee based on Eeex, but I don't see any public repo.

**Comment by alexbatalov** [comment]
I've briefly reviewed GemRB, your commitment during all these years is truly remarkable. It looks like you've followed "clean room" approach, as I see no signs of original symbols even in the very old commits (0.5.0). Given your goal it's a much better approach than mine as it reduces or even eliminates legal risks associated with reverse engineering. However, as I've already said it's a research project - I'm more interested in the code which powers the game rather than in the game itself. I don't have plans to support or otherwise improve recovered code beyond initial restoration (if it will ever be completed).

**Comment by lynxlynxlynx** [comment]
That's fine. If you succeed, it would be useful for us as well. There's so much stupid crap inside, you're lucky you have to contend with only one version. :)
